test/test_freelancer_dispute_resolver.py

```
@@ -0,0 +1,170 @@
+
+@pytest.mark.order(6)
+def test_non_freelancer_cannot_submit_deliverables(setup):
+    """Client should not be able to submit deliverables."""
+    try:
+        setup["client"].send_transaction(
+            sender=setup["client_account"],
+            contract_address=setup["contract_address"],
+            function="submit_deliverables",
+            args=["https://malicious-override.com"],
+        )
+        assert False, "Expected an exception but none was raised."
```

Ruff (0.15.1)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)